spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/support/SpringDocJsonDecodeFilterTest.java

```diff
@@ -56,8 +52,8 @@ public void doFilter(ServletRequest request, ServletResponse response) throws IO
                 super.doFilter(request, response);
             }
         };
-        springDocJsonDecodeFilter.doFilter(mockHttpServletRequest, mockHttpServletResponse, mockFilterChain);
-        assertEquals(OPENAPI_JSON_UNESCAPED_CONTENT, mockHttpServletResponse.getContentAsString());
+        new SpringDocJsonDecodeFilter().doFilter(mockHttpServletRequest, mockHttpServletResponse, mockFilterChain);
+        assertThat(OPENAPI_JSON_UNESCAPED_CONTENT).isEqualTo(mockHttpServletResponse.getContentAsString());
```

Comment:
**[TinyNit]** the assertion target should be the `actual` rather than the `expected`.

Reply:
done